tests/regression/test_solve_rhs.py

```diff
@@ -0,0 +1,25 @@
+from firedrake import *
+
+
+def test_solve_rhs():
+    mesh = UnitSquareMesh(8, 8, quadrilateral=True)
+    V = FunctionSpace(mesh, "CG", 1)
+    u1 = Function(V)
+    u2 = Function(V)
+    v = TestFunction(V)
+    f = Constant(1)
+
+    F1 = inner(grad(u1), grad(v))*dx - inner(f, v)*dx
+    F2 = inner(grad(u2), grad(v))*dx
+    rhs = assemble(inner(f, v)*dx)
+
+    bcs = DirichletBC(V, 0, "on_boundary")
```

Comment:
Might want to use non-homogeneous BCs for the test

Reply:
Agreed, done.